src/Repository/ReaderRepository.php

```diff
@@ -1,19 +1,27 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: ioana
- * Date: 11/28/2018
- * Time: 10:04 AM
- */
-include("DataBaseConnection.php");
 
+namespace BookReviews\Repository;
 
+/**
+ * Class ReaderRepository
+ * @package BookReview\Repository
+ */
 class ReaderRepository extends DataBaseConnection
 {
-     function InsertIntoDb($username, $password)
+    /**
+     * @param $username
```

Comment:
Missing type in comment. Proper format is `@param {type} {variable}`

Reply:
The `@param` annotation needs the typehint in it. In this situations the correct format is `@param string $username`